core/webpack.config.js

```diff
@@ -78,13 +86,24 @@ module.exports = {
     ]
   },
   plugins: [
-    new ExtractTextPlugin('luigi.css'),
+    new CleanWebpackPlugin(['public'], {
```

Comment:
Not sure if we should do that. Now each time I run `npm run bundle` when the app is running, the app compilation fails because there is no luigi.css file. So after bundling core, I have to re-run the app. Not the best developer experience :)